### PR TITLE
[Images] Fleshed out API request page

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

**Review by sejoker — APPROVED** [review]

File: content/images/cloudflare-images/api-request.md [hunk 1]
```diff
@@ -37,23 +37,50 @@ curl -X POST \
   -F file=@./<YOUR_IMAGE>
```
Comment (pedrosousa) [inline content/images/cloudflare-images/api-request.md]: Should this line also use the `<YOUR_IMAGE.IMG>` placeholder, with extension?

File: content/images/cloudflare-images/api-request.md [hunk 2]
```diff
@@ -37,23 +37,50 @@ curl -X POST \
...
+
+```json
+{
+  "result": {
+    "id": "<YOUR_IMAGE_ID>",
+    "filename": "<YOUR_IMAGE.IMG>",
+    "uploaded": "2021-09-14T05:52:14.767Z",
+    "requireSignedURLs": false,
+    "variants": [
+      "https://imagedelivery.net/<ACCOUNT_HASH>/<IMAGE_ID>/<VARIANT_NAME>"
```
Comment (pedrosousa) [inline content/images/cloudflare-images/api-request.md:52]: Does `<IMAGE_ID>` have the same value of `<YOUR_IMAGE_ID>` (line 47)? If it has, I would use the same placeholder name in both cases. There are several occurrences of this in the page.

File: content/images/cloudflare-images/api-request.md [hunk 3]
```diff
@@ -37,23 +37,50 @@ curl -X POST \
...
   "result_info": null,
   "success": true,
   "errors": [],
   "messages": []
 }
-```
\ No newline at end of file
+```
+
+You can use the `variants` URL from the response to [serve images](/images/cloudflare-images/serve-images/) from your Cloudflare Images account.
```
Comment (pedrosousa) [inline content/images/cloudflare-images/api-request.md]:
```suggestion
You can use the URLs in the `"variants"` property of the response to [serve images](/images/cloudflare-images/serve-images/) from your Cloudflare Images account.
```

**Review by pedrosousa — APPROVED** [review]

File: content/images/cloudflare-images/api-request.md (hunk 2 above)
Comment (marciocloudflare) [reply content/images/cloudflare-images/api-request.md:52]: Yes, it has. Good catch.

File: content/images/cloudflare-images/api-request.md (hunk 1 above)
Comment (marciocloudflare) [reply content/images/cloudflare-images/api-request.md]: Yup. Thanks!

**Pull request merged**